addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -318,17 +318,19 @@
                             <span class="mr-5 ml-auto">Public Views</span>
                         </span>
                     </div>
-                    <div class="col-lg-4 d-flex flex-column">
+                    <div t-if="slide.channel_id.allow_comment or slide.channel_id.channel_type == 'documentation'" class="col-lg-4 d-flex flex-column">
                         <span class="font-weight-bold mb-3 mt-3">Actions</span>
-                        <span class="d-flex pl-3">
-                            <span class="badge badge-pill" t-esc="slide.likes"/>
-                            <span class="mr-5 ml-auto">Likes</span>
-                        </span>
-                        <span class="d-flex pl-3">
-                            <span class="badge badge-pill" t-esc="slide.dislikes"/>
-                            <span class="mr-5 ml-auto">Dislikes</span>
-                        </span>
-                        <span class="d-flex pl-3">
+                        <t t-if="slide.channel_id.channel_type == 'documentation'">
+                            <span class="d-flex pl-3">
+                                <span class="badge badge-pill" t-esc="slide.likes"/>
+                                <span class="mr-5 ml-auto">Likes</span>
+                            </span>
+                            <span class="d-flex pl-3">
+                                <span class="badge badge-pill" t-esc="slide.dislikes"/>
+                                <span class="mr-5 ml-auto">Dislikes</span>
+                            </span>
+                        </t>
+                        <span t-if="slide.channel_id.allow_comment" class="d-flex pl-3">
```

Comment:
I think you can only filter on slide.channel_id.allow_comment in the parent div. and filter again on
slide.channel_id.channel_type == 'documentation' like it's already done. + remove this t-if here.